<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bg1.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: white
</string>
   </property>
   <widget class="QFrame" name="CardsFrame">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>871</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="card">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_8">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>190</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_10">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_12">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>370</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>370</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_13">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>370</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_14">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_15">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>190</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_16">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_17">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>190</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_18">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>370</y>
       <width>125</width>
       <height>170</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back_card.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="StatusBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#StatusBar{
color: white 
}

#PointsP1{
qproperty-alignment: AlignCenter;
background: teal;
border: 3px solid silver;
border-radius
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="GameName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CASINO ROYALE&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="timer">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TIMER: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="seconds">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;00:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentPlayer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;CurrentPlayer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QFrame" name="frameP1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="PointsP1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="player1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Player 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="txtPointsPlayer1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Points</string>
      </property>
     </widget>
     <widget class="QLabel" name="nickPlayer1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-style:italic;&quot;&gt;NickPlayer1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameP2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="txtPointsPlayer2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Points</string>
      </property>
     </widget>
     <widget class="QLabel" name="nickPlayer2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic;&quot;&gt;NickPlayer2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="player2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Player 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="PointsP2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
